src/store/car.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+// create thunk
+export const getCars = createAsyncThunk(
+  'cars/getCars',
+  async () => {
+    const getCarsData = await fetch('http://localhost:3000/api/v1/cars');
+    const res = await getCarsData.json();
+    return res;
+  },
+);
+
```

Comment:
**[Optional]** I suggest putting this block in the `car.service.js` file and then exporting it.